falcon/request.py

```diff
@@ -475,6 +480,59 @@ def headers(self):
     def params(self):
         return self._params
 
+    @property
+    def param(self):
+        if self._param is None:
+            self._param = helpers.ParamProxy(self, self.params)
+
+        return self._param
+
+    @property
+    def raw_body(self):
```

Comment:
That's an interesting alternative -- I think it could work pretty well. I agree that it's less than optimal to add more _stuff_ to the API surface if it doesn't serve a very clear use case; I wonder if it could be as simple as using a `BytesIO`?